valohai_yaml/generator/generator.py

```diff
@@ -0,0 +1,122 @@
+import os
```

Comment:
I'm not quite convinced this YAML generator code should live in `valohai-yaml` at all. 😕 (I'm sorry if I was unclear or incoherent in our balcony discussions about that.)

After all, it will only be able to generate configurations based on an AST that uses that as-of-yet-unreleased module that `valohai-yaml` is only a dependency of.